vllm/model_executor/model_loader/loader.py

```diff
@@ -736,15 +736,18 @@ def save_model(
 class BitsAndBytesModelLoader(BaseModelLoader):
     """Model loader to load model weights with BitAndBytes quantization."""
 
-    # TODO: these module names are for Llama only,
-    # change so that it works with other models as well
+    possible_config_file_names = ["adapter_config.json"]
+
     default_target_modules = [
-        "gate_proj", "down_proj", "up_proj", "q_proj", "k_proj", "v_proj",
-        "o_proj"
+        ".gate_proj.",
+        ".down_proj.",
+        ".up_proj.",
+        ".q_proj.",
+        ".k_proj.",
+        ".v_proj.",
+        ".o_proj.",
     ]
```

Comment:
For future consideration, some models like [Phi3](https://huggingface.co/microsoft/Phi-3.5-mini-instruct?show_file_info=model.safetensors.index.json) have merged modules in the HF checkpoint i.e. `qkv_proj`

Reply:
I added the known target modules to the list.